package.json

```diff
@@ -1,10 +1,12 @@
 {
   "dependencies": {
+    "@octokit/rest": "^20.0.2",
     "cors": "^2.8.5",
     "express": "^4.18.2",
     "firebase": "^10.5.2",
     "firebase-admin": "^11.11.0",
     "firebase-functions": "^4.5.0",
+    "octokit": "^3.1.2",
```

Comment:
is this import necessary? is the @octokit/rest above not sufficient?

Reply:
yeah I added both when I was trying to get it to work, I'll play around with it